src/main/java/liuni/models/TwitterTweetModel.java

```diff
@@ -0,0 +1,56 @@
+package liuni.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Date;
+
+public class TwitterTweetModel {
+    private String message;
+    private UserModel user;
+    private Date createdAt;
+
+    public TwitterTweetModel() {
+
+    }
+
+    @JsonProperty
+    public String getMessage() {
+        return message;
+    }
+
+    @JsonProperty
+    public UserModel getUser() {
+        return user;
+    }
+
+    @JsonProperty
+    public Date getCreatedAt() {
+        return createdAt;
+    }
+
+    @JsonProperty
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    @JsonProperty
+    public void setUser(UserModel user) {
+        this.user = user;
+    }
+
+    @JsonProperty
+    public void setCreatedAt(Date createdAt) {
+        this.createdAt = createdAt;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
if you override equals, what else do you need to override?

Reply:
Hashcode added, done.